src/FormContainer.tsx

```diff
@@ -118,6 +118,6 @@ const makeWrapper = <T extends {}>(config: IFormConfig) => (WrappedComponent: an
 };
 
 export const connectForm = <T extends {} = any>(
-    validators: any[] = [],
+    validators: ValidationRuleResult[] = [],
```

Comment:
Adding `const` to the `enum` allows me to do this. Previously it would throw an error.

Reply:
Actually, nope, nevermind...

```
TS4023: Exported variable 'connectForm' has or is using name 'ValidationType' from external module "/home/travis/build/vitkon/form-container/src/interfaces" but cannot be named.
```
I'll have to revert this. 😞